scenes/inventory/inventory.gd

```diff
@@ -4,3 +4,19 @@ class_name Inventory
 extends Resource
 
 @export var items: Array[InventoryItem]
+
+
+func clear() -> void:
+	items.clear()
+
+
+func has_item(item: InventoryItem) -> bool:
+	return item in items
+
+
+func add_item(item: InventoryItem) -> void:
+	items.push_back(item)
+
+
+func amount_of_items() -> int:
+	return items.size()
```

Comment:
These functions are not doing much. But I guess that you don't want to let the users of the Inventory resource access items directly.

Reply:
Yep, I didn't want to expose the inventory's internals